src/tests/functional/plugin/shared/include/behavior/ov_plugin/core_integration.hpp

```diff
@@ -1110,16 +1110,16 @@ TEST_P(OVClassLoadNetworkWithDefaultPropertiesTest, LoadNetworkWithDefaultProper
     OV_ASSERT_NO_THROW(model = ie.compile_model(actualNetwork, target_device, configuration));
     ov::hint::PerformanceMode value = ov::hint::PerformanceMode::UNDEFINED;
     OV_ASSERT_NO_THROW(value = model.get_property(ov::hint::performance_mode));
-    ASSERT_EQ(value, ov::hint::PerformanceMode::THROUGHPUT);
+    ASSERT_EQ(value, ov::hint::PerformanceMode::LATENCY);
 }
 
 TEST_P(OVClassLoadNetworkWithDefaultIncorrectPropertiesTest, LoadNetworkWithDefaultIncorrectPropertiesTest) {
     ov::Core ie = createCoreWithTemplate();
     ov::CompiledModel model;
     OV_ASSERT_NO_THROW(model = ie.compile_model(actualNetwork, target_device, configuration));
-    ov::hint::PerformanceMode value = ov::hint::PerformanceMode::THROUGHPUT;
+    ov::hint::PerformanceMode value = ov::hint::PerformanceMode::LATENCY;
```

Comment:
Should keep THROUGHPUT test case for MULTI. Create new test case for AUTO with default LATENCY.